spec/send_headers.spec.php

```diff
@@ -183,7 +183,7 @@
 
 			it('is in developer mode on the local dev environment', function () {
 				expect('get_post_types')->toBeCalled()->once();
-				expect('get_field')->toBeCalled()->times(2);
+				expect('get_field')->toBeCalled()->times(3);
```

Comment:
This looks good, but I think we are not specifically testing the case where `get_field()` returns `null`?